content-src/components/TopSites/TopSites.js

```diff
@@ -25,40 +60,37 @@ const TopSites = React.createClass({
       page: "NEW_TAB"
     };
   },
-  onClick(index) {
-    this.props.dispatch(actions.NotifyEvent({
-      event: "CLICK",
-      page: this.props.page,
-      source: "TOP_SITES",
-      action_position: index,
-      metadata_source: this.props.sites[index].metadata_source
-    }));
+  onClickFactory(index, site) {
+    return () => {
+      let payload = {
+        event: "CLICK",
+        page: this.props.page,
+        source: "TOP_SITES",
+        action_position: index,
+        metadata_source: site.metadata_source
+      };
+      if (site.recommended) {
```

Comment:
you can remove this recommendation stuff - top sites will never have a recommendation and also we removed the recommendation logic so this can go. (lines 72-79)